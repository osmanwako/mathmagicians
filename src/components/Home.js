function getMessage() {
  const str = `Sure! Magic Maths is a set of React components that allow you to create interactive and animated visualizations with declarative code. It is a great tool for creating educational content, games, and simulations that involve mathematical concepts. You can use it to create graphs, charts, animations, and more. The components are easy to use and customizable, and they come with a wide range of features such as zooming, panning, and animation controls`;
  return str;
}

function Home() {
  return (
    <div>
      <h3>Welcome to Our page</h3>
      <p>
        {getMessage()}
      </p>
      <p>
        {getMessage()}
      </p>
    </div>
  );
}

export default Home;
