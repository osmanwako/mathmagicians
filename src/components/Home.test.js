import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

test('Home component renders correctly', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
});
