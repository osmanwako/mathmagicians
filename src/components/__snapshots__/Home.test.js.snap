// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
<div>
  <h3>
    Welcome to Our page
  </h3>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Reprehenderit quasi laborum quisquam eum ipsam iusto, animi blanditiis aperiam nobis laudantium dolore quas cumque molestias,a perferendis corporis optio repudiandae quia ?
          Lorem ipsum dolor, sit amet consectetur adipisicing elit.Quos quidem iure illo suscipit voluptatum non at laudantium! Eum voluptatibus doloremque voluptate deleniti odio
      natus qui ex tempore error, dignissimos nostrum!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Reprehenderit quasi laborum quisquam eum ipsam iusto, animi blanditiis aperiam nobis laudantium dolore quas cumque molestias,a perferendis corporis optio repudiandae quia ?
          Lorem ipsum dolor, sit amet consectetur adipisicing elit.Quos quidem iure illo suscipit voluptatum non at laudantium! Eum voluptatibus doloremque voluptate deleniti odio
      natus qui ex tempore error, dignissimos nostrum!
  </p>
</div>
`;
